src/locales/zh.json

```
@@ -412,7 +412,11 @@
       "SearchPlaceholder": "Search Superhero",
       "SortingLatest": "最新的",
       "SortingHighestRated": "获取♥币最高的 记录排序",
-      "SortingMostPopular": "最受欢迎"
+      "SortingMostPopular": "最受欢迎",
+      "Clear": "Clear",
+      "CloseSearch": "Close Search",
+      "ShowFiltered": "Show filtered content",
+      "HideFiltered": "Hide filtered content"
```

I think the best is to use the same text for both states
```suggestion
      "ShowFiltered": "Filter content",
      "HideFiltered": "Filter content"
```
or may be "Safe content only" could also be a good title